src/features/scheduler/components/MonthView.tsx

```diff
@@ -40,21 +40,18 @@ export const MonthView = ({ userEvents }: { userEvents: UserEvent[] }) => {
   const [isEdit, setIsEdit] = useState(false);
   const { user } = useContext(AuthContext);
 
-  const { displayedYear, displayedMonth, calendarViewTab, shouldUpdateData } = useAppSelector(
-    ({ yearsReducer }) => yearsReducer
-  );
+  const { displayedYear, displayedMonth, calendarViewTab, shouldUpdateData } =
+    useAppSelector(({ yearsReducer }) => yearsReducer);
   const { setShouldUpdateData } = yearsSlice.actions;
   const dispatch = useDispatch();
 
   useEffect(() => {
```

Comment:
useMemo maybe?

Reply:
good idea

Reply:
I have reviewed that code and it looks like impossible, couse I update allDays in a different place